src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Button } from '../Buttons';
+import { Modal, ModalProps } from '../Modal';
+import styles from './Alert.scss';
+
+interface Props extends ModalProps {
+    /** Should the alert appear */
+    showAlert?: boolean;
+    /** The alert content */
+    children: ReactNode;
+    /** Dialog title */
+    title?: string;
+    /** Callback to be called on Accept button click (shown in both alert and confirm) */
+    onAccept: () => void;
+    /** Callback to be called on Cancel button click (only relevant for confirm dialog) */
+    onCancel: () => void;
+    /** Accept button's text */
+    acceptLabel: string;
+    /** Cancel button's text */
+    cancelLabel: string;
```

Comment:
I think it makes sense to group button's props in one `Button` type. In this case, we always have a label and callback for the button. We can make  `cancelButton` optional.
```
type Button = {
    label: string;
    onClick: () => void;
}
.....
acceptButton: Button;
cancelButton?: Button;
```